src/main/java/io/reactivex/internal/schedulers/ExecutorScheduler.java

```diff
@@ -330,6 +332,12 @@ public void dispose() {
                 direct.dispose();
             }
         }
+
+        @Override
+        public Runnable getWrappedRunnable() {
+            Runnable r = get();
+            return r != null ? r : Functions.EMPTY_RUNNABLE;
```

Comment:
Alternative solution could be to throw `IllegalStateException` I guess

Reply:
This is not a failure state but a terminal state of the wrapper. Penalizing it would be too harsh.

Reply:
For this particular introspection api it seems like an invalid state, but I'm ~fine with current solution.

The problem of returning empty runnable is that since it's the same object every time — it can mess up introspection that is based on object identity/equality